(DOCSP-27022): Release notes for December 14, 2022 (#265)
mongodb/docs-app-services: source/release-notes/backend.txt

2022 Releases
-------------

.. _backend_20221214:

14 December 2022 Release
~~~~~~~~~~~~~~~~~~~~~~~~

> TODO: see what ref will be for custom expiration times, and add.

Author: ```suggestion
TODO: waiting on link for Admin API endpoints for changing expiration times. will add when ready. 
```

- Raise :ref:`Atlas Function request timeout <function-constraints>`
  from 150 seconds to 180 seconds.

> - Let users configure expiration times for refresh tokens.

Author: ```suggestion
- Let users configure expiration times for refresh tokens from the Admin API.
```

> - Let users configure expiration times for refresh tokens from the Admin API.

Reviewer: instead of using language like 'Let users' can we just say 'Expiration times for refresh tokens can be configured from the API'

Author: i get how this sounds funny. done b/c we try to avoid passive voice in the docs. i'll use your phrasing.

Author: ```suggestion
- Expiration times for refresh tokens can be configured from the API.
```

> - Let users create Apps in AWS region Ohio (us-east-2) from the UI.

Reviewer: this is in prev release notes, maybe we just want to move it up to be accurate?

Author: in prev release notes, it was enabled on backend. but this time in the UI as well.
